R/report-plots.R

```diff
@@ -360,8 +360,12 @@ bar_plotly <- function(df,
   } else if (ind == "art_coverage") {
     title <- "ART coverage"
   }
+
+  age_groups <- get_age_groups()
+  age_label <- age_groups[age_groups$age_group == unique(plot_data$age_group), ]$age_group_label
+
   plot_title <- paste("<b>", title, ": ", "Household survey vs. ",
```

Comment:
Is this doing the spacing you want? `paste` adds a ` ` in between each arg. If you want to add the spaces yourself manually you can use `paste0` to not add a separator. Possible that the markdown fixes any weird spacing for you. I think at the end you also want a closing tag `</b>`

Reply:
Thanks - have updated.